command/credits.go

```diff
@@ -0,0 +1,248 @@
+package command
+
+import (
+	"bytes"
+	"fmt"
+	"math"
+	"math/rand"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+
+	"github.com/cli/cli/utils"
+)
+
+var isWindows bool
+
+var thankYou = `
+     _                    _
+    | |                  | |
+_|_ | |     __,   _  _   | |           __
+ |  |/ \   /  |  / |/ |  |/_)   |   | /  \_|   |
+ |_/|   |_/\_/|_/  |  |_/| \_/   \_/|/\__/  \_/|_/
+                                   /|
+                                   \|
+                              _
+                           o | |                           |
+ __   __   _  _  _|_  ,_     | |        _|_  __   ,_    ,  |
+/    /  \_/ |/ |  |  /  |  | |/ \_|   |  |  /  \_/  |  / \_|
+\___/\__/   |  |_/|_/   |_/|_/\_/  \_/|_/|_/\__/    |_/ \/ o
+
+
+`
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+	isWindows = runtime.GOOS == "windows"
+
+	RootCmd.AddCommand(creditsCmd)
+
+	creditsCmd.Flags().BoolP("static", "s", false, "Print a static version of the credits")
```

Comment:
Perhaps instead of "static" in the description, we can say "Print a non-animated version of the credits"? If a person hasn't run `credits` yet, they might not realize that the default is animated, so it would be puzzling as to what "static" means